backend/app/views/spree/admin/users/index.html.erb

```diff
@@ -89,7 +89,7 @@
           <% if can?(:edit, user) %>
             <%= link_to_edit user, no_text: true %>
           <% end %>
-          <% if can?(:destroy, user) %>
+          <% if can?(:destroy, user) && user.orders.count.zero? %>
```

Comment:
what is the advantage of doing `user.orders.count.zero? ` instead of `user.orders.empty?`

Reply:
None, but since both snippets generate a SQL query, they're pretty much exchangeable. Your suggestion is less verbose and feels more natural, though.